### PR TITLE
Feature/notification sending priority

### PR DESCRIPTION
**Pull request by rubvardanyan**

### CODE REVIEW

File: core/core-db-entities/src/main/java/com/sflpro/notifier/db/entities/notification/NotificationSendingPriority.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.sflpro.notifier.db.entities.notification;
+
+/**
+ * User: Ruben Vardanyan
+ * Company: SFL LLC
+ * Date: 07/30/2021
+ * Time: 11:22
+ */
+public enum NotificationSendingPriority {
```
Comment (mikron) [inline core/core-db-entities/src/main/java/com/sflpro/notifier/db/entities/notification/NotificationSendingPriority.java:9]: Pity we don't have commons module

File: core/core-db-repositories/src/main/resources/db/migration/mysql/V1_13__Notification_sending_priority.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+alter table notification
+    add column sending_priority varchar(50) null;
+
+update notification
```
Comment (mikron) [inline core/core-db-repositories/src/main/resources/db/migration/mysql/V1_13__Notification_sending_priority.sql:4]: This one, maybe will cause issues on the container startup, for big projects (fintech) for example

**Review by mikron — CHANGES_REQUESTED** [review]

File: api/api-model/src/main/java/com/sflpro/notifier/api/model/notification/NotificationSendingPriorityClientType.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * User: Ruben Vardanyan
+ * Company: SFL LLC
+ * Date: 07/30/2021
+ * Time: 12:23
+ */
+public enum NotificationSendingPriorityClientType {
+    LOW,
+    NORMAL,
```
[the remark this answers is not in the document]
Comment (rubvardanyan) [reply api/api-model/src/main/java/com/sflpro/notifier/api/model/notification/NotificationSendingPriorityClientType.java]: Renamed

File: core/core-db-repositories/src/main/resources/db/migration/mysql/V1_13__Notification_sending_priority.sql (hunk 2 above)
Comment (rubvardanyan) [reply core/core-db-repositories/src/main/resources/db/migration/mysql/V1_13__Notification_sending_priority.sql:4]: Will try to test the migrations for MySQL with generated data.

**Review by mikron — APPROVED** [review]

**Pull request merged**